go.mod

```diff
@@ -1,14 +1,16 @@
 module github.com/cometbft/cometbft
 
-go 1.22
+go 1.22.5
+
+toolchain go1.23.0
```

Comment:
any particular reason we need to force the `1.23` toolchain for building the project ?

Reply:
Not really. `go mod tidy` adds it automatically